voltaire_bundler/bundle/bundle_manager.py

```
@@ -574,6 +601,19 @@ async def remove_included_and_readd_to_mempool_userops_monitoring(
+                # the background so the next client poll for this userop
+                # finds them hot instead of paying two more RPC round trips.
+                # The log entry's transactionHash is the only thing we need.
+                log_entry = user_operation_log[0]
+                if "transactionHash" in log_entry:
+                    asyncio.create_task(
+                        _warm_inclusion_caches(
+                            self.ethereum_node_urls,
+                            local_mempool.user_operation_handler,
+                            log_entry["transactionHash"],
+                        )
+                    )
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)